tests/test_text_processing.py

```diff
@@ -56,3 +57,20 @@ def test_move_span_ids_to_sections():
 <section class="section" data-type="sect2" id="sec-biastypes">
 <h2>Types of Bias</h2>
 <p>Bias comes in many forms!</p></section>"""
+
+
+def test_sidebar_processing():
+    """
+    Sidebars from jupyter book (from the ```{sidebar} syntax) are
+    formatted as <aside>s with a "sidebar" class. For HTMLBook, these
+    need to have sidebar data-types, and the paragraph with the
+    "sidebar-title" class should be an <h5> element.
+    """
+    chapter_text = BeautifulSoup("""<aside class="sidebar">
+<p class="sidebar-title">Here Is a Sidebar Title</p>
+<p>And this is some sidebar content!</p>
+</div>
+</aside>""", "html.parser")
```

Comment:
Just curious, would it be worthwhile to have this example include a non-sidebar aside (or non-aside content, for that matter) to verify that it doesn't get processed in the same way?

Reply:
To me, that would be testing Beautiful Soup, since we're using their filtering to find the asides for processing (`sidebars = chapter.find_all("aside", class_="sidebar")`. Happy to be wrong about that though.